tests/hypothesis_strategies/kernels.py

```diff
@@ -124,13 +127,13 @@ def single_kernels(draw: st.DrawFn):
 
 @st.composite
 def kernels(draw: st.DrawFn):
-    """Generate :class:`baybe.kernels.basic.Kernel`."""
+    """Generate :class:`baybe.kernels.base.Kernel`."""
     kernel_type = draw(st.sampled_from(KernelType))
 
     if kernel_type is KernelType.SINGLE:
         return draw(single_kernels())
 
-    base_kernels = draw(st.lists(single_kernels()))
+    base_kernels = draw(st.lists(single_kernels(), min_size=2))
```

Comment:
Shouldn't we also test the case of trying to have an `AdditiveKernel` that only has a single kernel for stability reasons? Or do we explicitly exclude this case?

Reply:
We exclude it via a validator in the kernel class itself, because it simply brings no benefit, so all good here 👍🏼